rules/aws_cloudtrail_rules/aws_ec2_vulnerable_xz_image_launched.py

```diff
@@ -0,0 +1,71 @@
+from panther_base_helpers import aws_rule_context
+from panther_default import aws_cloudtrail_success
+
+# AMIs published by Fedora between 2022-03-26 and 2022-04-02
+# OpenSUSE and Kali do not have any recent [public] AMIs that would be affected
+
+# pylint: disable=line-too-long
+AMIS = {
```

Comment:
Should we put this into a LUT instead given that AMIs are region-specific?

Reply:
Ah, yea, probably. I didn't even think about that.